tsconfig.json

```diff
@@ -4,55 +4,66 @@
 
     /* Basic Options */
     // "incremental": true,                   /* Enable incremental compilation */
-    "target": "es5",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
-    "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */
-    "lib": ["es2018", "esnext.asynciterable", "dom"],                             /* Specify library files to be included in the compilation. */
-    // "allowJs": true,                       /* Allow javascript files to be compiled. */
+    "target": "es5" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */,
```

Comment:
Mulig det bare er meg som syns denne filen er vanskelig å lese med kommentarene, men hadde den ikke vært mye ryddigere uten alle kommentarene om mulige valg?

Jeg føler at kommentarene gir lite verdi (for meg i alle fall siden om jeg skal endre på noe her så ville jeg slått opp hva settingen faktisk gjør i doc'en uansett).

(u)enig?